CHANGELOG.md

```diff
@@ -3,10 +3,15 @@
 ## 1.5.0 (TBD)
 
 ### New Features
+
 - Added support for new functions in `snowflake.snowpark.functions`:
   - `array_generate_range`
   - `sequence`
 
+### Bug Fixes
+
+- Fixed type hint declaration for custom types - `ColumnOrName`, `ColumnOrLiteralStr`, `ColumnOrSqlExpr`, `LiteralType` and `ColumnOrLiteral` that were breaking `mypy` checks. Contributor: @StefanUlbrich.
+
```

Comment:
```suggestion
- Fixed type hint declaration for custom types - `ColumnOrName`, `ColumnOrLiteralStr`, `ColumnOrSqlExpr`, `LiteralType` and `ColumnOrLiteral` that were breaking `mypy` checks.

```

Reply:
May I ask why?

Reply:
@StefanUlbrich sorry, this ask came from me.

1. If you look at our [CHANGELOG.md](https://github.com/snowflakedb/snowpark-python/blob/main/CHANGELOG.md) you'll see that we don't usually give shoutouts. So I believe that it'd be unfair to people that contributed previously. So unless we do the shoutouts every time I don't think we should add them at all.
2. I believe that changelogs should be concise for users to read. Its main purpose is to see if a user needs to dive into repo in deeper detail.

Please note that the commit will still be tied to your account. It's only the changelog shoutout that I asked to be removed mostly because of reason 1.

Reply:
Sounds reasonable. Thanks for the clarification 🙂